tests/integration/conftest.py

```diff
@@ -374,3 +374,124 @@ def create_nodebalancer_with_default_conf():
     res_arr = result.split(",")
     nodebalancer_id = res_arr[0]
     delete_target_id(target="nodebalancers", id=nodebalancer_id)
+
+
+@pytest.fixture
+def test_vpc_wo_subnet():
+    region = get_regions_with_vpcs_capabilties()[0]
+
+    label = str(time.time_ns()) + "label"
+
+    vpc_id = (
+        exec_test_command(
+            [
+                "linode-cli",
+                "vpcs",
+                "create",
+                "--label",
+                label,
+                "--region",
+                region,
+                "--no-headers",
+                "--text",
```

Comment:
I wonder if it'd make sense to start using `--json` and parsing the output just to make sure we're not affected by any `--text` output changes.

Reply:
That's a good point and it will be good in the long run like you said. I will create few tickets to use `--json` instead of `--text` in the integration test suites, there are quiet a number of functions/fixtures that use `--text` and `--delimiter`